python/taichi/cc_compose.py

```diff
@@ -117,6 +117,23 @@ def do_compile_kernel(self, e):
         declaration = source.split('{', 1)[0].strip()
         self.emit_header(f'extern {declaration};')
 
+    def do_config(self, e):
+        key = e['key']
+        value = e['value']
+        variable = f'Ti_cfg_{key}'
+        if isinstance(value, str):
+            declaration = f'const char {variable}[]'
+            value = repr(value)
+            value[0] = '"'
+            value[-1] = '"'
```

Comment:
Please note that `str` object cannot be assigned in this way. Maybe using slice is a better way. : )
```suggestion
            value = '\"' + value[1:-1] + '\"'
```